api/src/pcapi/routes/serialization/public_api_collective_offers_serialize.py

```diff
@@ -185,10 +185,48 @@ class CollectiveOffersStudentLevelResponseModel(BaseModel):
     name: str
 
 
+MAX_LIMIT_EDUCATIONAL_INSTITUTION = 20
+
+
+class GetListEducationalInstitutionsQueryModel(BaseModel):
+    id: int | None
+    name: str | None
+    institution_type: str | None
+    city: str | None
+    postal_code: str | None
+    limit: int = MAX_LIMIT_EDUCATIONAL_INSTITUTION
+
+    @validator("limit")
+    def validate_limit(  # pylint: disable=no-self-argument
+        cls: "GetListEducationalInstitutionsQueryModel", limit: int
+    ) -> int:
+        limit = min(limit, MAX_LIMIT_EDUCATIONAL_INSTITUTION)
+        return limit
+
+    class Config:
+        alias_generator = to_camel
+        extra = "forbid"
+
+
 class CollectiveOffersListStudentLevelsResponseModel(BaseModel):
     __root__: list[CollectiveOffersStudentLevelResponseModel]
 
 
+class CollectiveOffersEducationalInstitutionResponseModel(BaseModel):
+    id: int
+    name: str
+    institutionType: str
```

Comment:
attention je crois que certains peuvent être nulls :/ des cas particuliers un peu relou

Reply:
seulment institutionType est pas noté en not Null mais en pratique il n'y en a pas de null (on devrait pouvoir rajouter la contrainte) du coup non, aucun de ces champs ne peut être null